src/screens/VaultScreen.js

```diff
@@ -153,12 +199,25 @@ export default class VaultScreen extends React.Component {
               <Text style={{color:'white', fontSize:12, fontFamily: 'source-sans-pro-semibold',}}>OPEN RESOURCE</Text>
             </View>
             <View style={ styles.upvote }>
-              <TouchableHighlight onPress={() => { this.upvote(item.id); }}>
-                <Text style={ styles.upvoteText }>{ item.upvotes }</Text>
+              <TouchableHighlight onPress={() => { this.upvote(item.id, item.veteran_has_upvoted); }}>
```

Comment:
Remind me to refactor these rocket functions later with a partial. Iirc this is not the best way to do it since the function gets recreated at every render but I’m also doing this too cause lazy